src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
That's kind of a idealistic policy, imo. It seems to put the goals of the Debian project ahead of the software they're integrating, i.e. that software should work without any configuration or changes to the environment. By comparison, Red Hat has `/etc/profile.d` and they manage to struggle along (https://bugs.launchpad.net/ubuntu/+source/base-files/+bug/18808). Most of the time that's fine, but in this case it makes the tool (`rospkg`) unable to function outside of `/usr`, which kind of sucks. The policy's proposed work around of replacing all commands with a shell wrapper script would, I think, introduce a noticeable delay for commands used in tab-completion. And what about software that provides shell functions or tab completion helpers? They don't work for all shells (which was one of the justifications of the policy), how are they different from setting an env variable (which is extremely portable btw)?

Given that the policy is not going to change, then I'd still opt for having the normal setup files and let users source them and/or add them to their shell rc files as they like. This behavior (of always having `/usr/share` in the RPP) is unfortunately a deal breaker for using the debian version of `rospkg` with ROS installed to `/opt`.